src/Payloads/TableV2Payload.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace LaraDumps\LaraDumps\Payloads;
+
+use LaraDumps\LaraDumps\Concerns\Converter;
+
+class TableV2Payload extends Payload
+{
+    use Converter;
+
+    /** @var array */
+    protected $values;
+
+    /** @var string */
+    protected $label;
+
+    /** @var null|string */
+    protected $type = null;
+
+    public function __construct(array $values, string $label = 'Table', string $type = 'table-v2')
```

Comment:
Please use Constructor Promotion: https://www.php.net/manual/en/language.oop5.decon.php#language.oop5.decon.constructor.promotion

Reply:
ohk... will convert it.